pkg/connector/ble/device_windows.go

```diff
@@ -0,0 +1,10 @@
+package ble
+
+import (
+	"errors"
+	"github.com/go-ble/ble"
+)
+
+func newDevice() (ble.Device, error) {
+	return nil, errors.New("Not supported on Windows")
```

Comment:
For consistency with other Golang packages, error strings should start with a lower-case letter ("not" instead of "Not").